File: 2527-find-xor-beauty-of-array/2527-find-xor-beauty-of-array.java
class Solution {
    public int xorBeauty(int[] nums) {
        int xored = ((nums[0] | nums[0]) & nums[0]);
        int n = nums.length;
        if(n == 1) return xored;
        for(int i = 1; i < n; ++i)
            xored ^= nums[i];
        return xored;
    }
}

File: 2527-find-xor-beauty-of-array/2527-find-xor-beauty-of-array.py
class Solution:
    def xorBeauty(self, nums: List[int]) -> int:
        xored, n = (nums[0] | nums[0]) & nums[0], len(nums);
        if n == 1: return xored
        for i in range (1, n):
            xored ^= nums[i]
        return xored